src/utils/RealtimeFeedUtilsV3.js

```diff
@@ -0,0 +1,150 @@
+// @flow
+import { PYTHON_APP } from './EnvUtils';
+import Constants from './Constants';
+import LogUtils from './LogUtils';
+import RequestUtils from './RequestUtils';
+
+async function fetchRTF(): Object {
+  const options = {
+    ...Constants.GET_OPTIONS,
+    url: `http://${PYTHON_APP || 'localhost'}:5000/rtf`,
+  };
+  const data = await RequestUtils.createRequest(options, 'RTF request failed');
+  return JSON.parse(data);
+}
+
+async function fetchVehicles(): Object {
+  const options = {
+    ...Constants.GET_OPTIONS,
+    url: `http://${PYTHON_APP || 'localhost'}:5000/vehicles`,
+  };
+  const data = await RequestUtils.createRequest(options, 'Vehicles request failed');
+  return JSON.parse(data);
+}
+
+/**
+ * Given an array of { routeID, tripID },
+ * Return bus information
+ * Input:[
+ * {
+ * routeID : String,
```

Comment:
is this routeID capitalized because that's what frontend stores?
why are we being inconsistent with their naming?

Reply:
i used to use routeID last semester because i didn't think naming mattered until i realized that there were inconsistencies everywhere!!!

like i said, i'm leaving updating the documentation to another PR, too overwhelming if i also updated documentation in this one